config/delta-producer/public/export.json

```diff
@@ -63,13 +65,13 @@
     {
       "type": "http://www.w3.org/ns/org#Site",
       "graphsFilter": [
-        "http://mu.semte.ch/graphs/organisatieportaal"
+        "http://mu.semte.ch/graphs/worship-service",
```

Comment:
No risk to have sites in the shared graph ? in the previous producer the shared graph was also specified